data/soapvalidator/MailClient/HAB/ZCS-5705_ZimbraHABOu.xml

```
@@ -0,0 +1,341 @@
+		</t:objective>
+		<t:steps>
+			1. Fire HabOrgUnitRequest for an OU already deleted
+			2. Verify
+			error is received in the response
+		</t:steps>
+
+		<t:test>
+			<t:request>
+				<HABOrgUnitRequest op="rename" name="${ou1.newname}"
+					newName="${ou5.newname}" xmlns="urn:zimbraAdmin">
+					<domain by="name">${domain.name}</domain>
+				</HABOrgUnitRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//zimbra:Code" match="service.FAILURE" />  <!-- Should fail -->
+			</t:response>
+		</t:test>
+
+	</t:test_case>
+
+
+	<t:test_case testcaseid="ZimbraHABCreateOu_9" type="smoke"
+		bugids="ZCS-5705">
+		<t:objective>ForceDelete should be an optional parameter and work with
+			non delete operation
+		</t:objective>
+		<t:steps>
+			1. HabOrgUnitRequest for force delete operation with create
+			OU request
+			2. Verify success is received in the response
+		</t:steps>
+
+		<t:test>
+			<t:request>
+				<HABOrgUnitRequest op="create" name="${ou6.name}"
+					forceDelete="1" xmlns="urn:zimbraAdmin">
+					<domain by="name">${domain.name}</domain>
+				</HABOrgUnitRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:HABOrgUnitResponse//admin:habOrgUnitName"
+					match="${ou6.name}" />
+			</t:response>
+		</t:test>
+
+	</t:test_case>
+
+	<!-- Deprecated as forceDelete has not been implemented -->
+	<t:test_case testcaseid="ZimbraHABCreateOu_10" type="deprecated"
+		bugids="ZCS-5705">
+		<t:objective>ForceDelete should only work with delete operation
+		</t:objective>
+		<t:steps>
+			1. HabOrgUnitRequest for force delete operation for delete OU
+			request
+			2. Verify response as success
+		</t:steps>
+
+		<t:test>
+			<t:request>
+				<HABOrgUnitRequest op="delete" name="${ou3.name}"
+					forceDelete="1" xmlns="urn:zimbraAdmin">
+					<domain by="id">${domain.id}</domain>
+				</HABOrgUnitRequest>
+			</t:request>
+			<t:response>
+				<t:select path="//admin:HABOrgUnitResponse" />  <!-- Should pass -->
+			</t:response>
```

This should throw an error saying forceDelete is not supported.